CHANGES

```diff
@@ -1,4 +1,4 @@
-Changes from 8.8.2 to 8.9+beta1
+Changes from 8.8.1 to 8.9+beta1
```

Comment:
Traditionally, this section was called "Changes beyond 8.8" to avoid anticipating too much (and was renamed at release time). Now, a 8.8.2 should indeed likely happen so this section title will likely reveal accurate.